lua/wire/client/pickcolor_browser.lua

```diff
@@ -0,0 +1,109 @@
+--[[
+	
+	ColorPicker for Expression2
+	Allows faster color picking whithin the editor.
+	
+	by NickMBR
+	Dec 2016
+	
+]]--
+
+-- Convars
+CopyWithVecString = CreateClientConVar("wire_expression2_colorwithvec", 1, true, false)
+CopyWithAlpha = CreateClientConVar("wire_expression2_copyalpha", 0, true, false)
+
+-- Follows sound_browser pattern
+-- Sets up the panel
+local PickColorPanel = nil
+
+-- Checks the Color and Checkboxes and copies to clipboard.
+local function ColorToClipboard(colorStr)
+	
+	local clrCopy
+	
+	if (CopyWithVecString:GetInt() == 1 && CopyWithAlpha:GetInt() == 1) then
+		clrCopy = "vec4("..colorStr.r..","..colorStr.g..","..colorStr.b..","..colorStr.a..")"
+		
+	elseif (CopyWithVecString:GetInt() == 1 && CopyWithAlpha:GetInt() == 0) then
+		clrCopy = "vec("..colorStr.r..","..colorStr.g..","..colorStr.b..")"
+		
+	elseif (CopyWithVecString:GetInt() == 0 && CopyWithAlpha:GetInt() == 1) then
+		clrCopy = colorStr.r..","..colorStr.g..","..colorStr.b..","..colorStr.a
+		
+	elseif (CopyWithVecString:GetInt() == 0 && CopyWithAlpha:GetInt() == 0) then
+		clrCopy = colorStr.r..","..colorStr.g..","..colorStr.b
+	end
+	
+	-- Copy it!
+	if (clrCopy != nil) then SetClipboardText(clrCopy) end
+end
+
+-- Creates the Color Panel
+local function CreatePickColorBrowser()
+	if(IsValid(PickColorPanel)) then PickColorPanel:Remove() end
+	
+	-- Default
+	PickColorPanel = vgui.Create("DFrame")
+	PickColorPanel:SetPos(50,25)
+	PickColorPanel:SetSize(300, 340)
+
+	PickColorPanel:SetMinWidth(700)
+	PickColorPanel:SetMinHeight(400)
```

Comment:
Are these `SetMinWidth` / `SetMinHeight` calls necessary (given that the code sets the size above and makes it unresizable below)?